kong.conf.default

```diff
@@ -269,6 +269,11 @@
                                  # Similarly to `error_template_html`, the template
                                  # is required to contain one single `%s` placeholder for
                                  # the error message.
+
+#worker_events_max_payload =     # Specifies the max payload size the `worker_events` can
```

Comment:
`worker_events_max_payload_size` recommanded.

Reply:
I gave the name referring to the parameter `cluster_max_payload` which is the existed one defining the size of a  payload, do you think if it's better to keep the shape of the line?

Reply:
sound reasonable.